cfy_manager/components/nginx/nginx.py

```diff
@@ -265,6 +278,17 @@ def _external_certs_exist():
             else:
                 return False
         else:
+            # cert not provided - check the current cert: if it exists,
+            # and if it was autogenerated - does it still both ips
```

Comment:
I think you a word here.

"does it still both ips"?

Reply:
accidentally a word